### DISCUSSION TITLE
How to set the shm_size for a container via the GUI?

### DISCUSSION THREAD

**Discussion by zariok** [podman-desktop/podman-desktop#5332] [Q&A]
While attempting to use linuxserver/docker-rdesktop entries, the chrome browser was failing to render sites when starting from the GUI.   The solution is to set shm_size=2gb via CLI.  

```podman run --shm-size=2gb lscr.io/linuxserver/rdesktop:fedora-kde```

Is there a method to pass additional arguments to podman via the GUI?

**Comment by deboer-tim** [comment]
Sorry, I don't think there's a way to set shm size nor enter unknown args in Podman Desktop today. Please feel free to open up an enhancement request issue.
